tests/unit/SidebarFilterTest.js

```diff
@@ -71,7 +72,59 @@ describe('Sidebar', () => {
             );
         });
 
-        it('includes or excludes policy expense chats depending on the beta', () => {
+        it('excludes an empty chat report', () => {
+            LHNTestUtils.getDefaultRenderedSidebarLinks();
+
+            // Given a new report
+            const report = LHNTestUtils.getFakeReport(['emptychat+1@test.com', 'emptychat+2@test.com'], 0);
+
+            return (
+                waitForPromisesToResolve()
+                    // When Onyx is updated to contain that report
+                    .then(() =>
+                        Onyx.multiSet({
+                            [`${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`]: report,
+                        }),
+                    )
+
+                    // Then no reports are rendered in the LHN
+                    .then(() => {
+                        const hintText = Localize.translateLocal('accessibilityHints.chatUserDisplayNames');
+                        const displayNames = screen.queryAllByLabelText(hintText);
+                        expect(displayNames).toHaveLength(0);
+                    })
+            );
+        });
+
+        it('includes an empty chat report if it has a draft', () => {
+            LHNTestUtils.getDefaultRenderedSidebarLinks();
+
+            // Given a new report
+            const report = {
+                ...LHNTestUtils.getFakeReport([1, 2], 0),
+                hasDraft: true,
+            };
+
+            return (
+                waitForPromisesToResolve()
+                    // When Onyx is updated to contain that report
+                    .then(() =>
+                        Onyx.multiSet({
+                            [`${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`]: report,
+                            [ONYXKEYS.PERSONAL_DETAILS_LIST]: LHNTestUtils.fakePersonalDetails,
+                        }),
+                    )
+
+                    // Then no reports are rendered in the LHN
```

Comment:
Probably copy / pasta error? Should be "The report is rendered in the LHN"

Reply:
Oh yes, thks for catching that!

I'll create a P.R. to fix that unit test that is failing + fix that comment.